aspnetcore/fundamentals/configuration.md

```diff
@@ -341,11 +341,12 @@ results in an exception. An exception will be thrown if you specify a command-li
 
   ## The *web.config* file
 
-*web.config* is required when you host the app in IIS or IIS-Express. It turns on the AspNetCoreModule in IIS to launch your app. It may also be used to configure other IIS settings and modules. If you are using Visual Studio and delete *web.config*, Visual Studio will create a new one.
+*web.config* is required when you host the app in IIS or IIS-Express. *web.config* turns on the AspNetCoreModule in IIS to launch your app. *web.config* can also be used to configure other IIS settings and modules. If you are using Visual Studio and delete *web.config*, Visual Studio will create a new one.
```

Comment:
*web.config* is required --> A *web.config* file is required


*web.config* turns on...--> Settings in *web.config* enable the AspNetCoreModule in IIS to launch your app and configure other IIS settings and modules.

I'd avoid starting a sentence with lower-case "web.config"